frontend/src/services/web3-provider/metamask-provider.ts

```diff
@@ -34,7 +34,7 @@ export class MetaMaskProvider extends EthereumProvider implements ISigner {
     try {
       const accounts: string[] = await this.web3Provider.send('eth_requestAccounts', []);
       this.signer.value = this.web3Provider.getSigner(accounts[0]);
-      this.signerAddress.value = accounts[0];
+      this.signerAddress.value = getAddress(accounts[0]);
```

Comment:
Previously, addresses were stored here as lowercased strings instead of proper checksumed addresses which explains the usage of `getAddress` here.